package.json

```diff
@@ -49,6 +49,7 @@
   },
   "dependencies": {
     "@babel/runtime": "7.14.6",
+    "@patternfly/patternfly": "^4.96.2",
```

Comment:
if yah can lock down the version n add the lockfile travis will 🤗  yah

Reply:
@AllenBW I'm not sure what happened with the package.lock file...